plugins/web/opentelemetry-plugin-react-load/test/reactLoad.test.ts

```diff
@@ -208,10 +237,303 @@ import { AttributeNames } from '../src/enums/AttributeNames';
+              'render span is not a child of the updating span'
+            );
+            assert.equal(
+              getSnapshotBeforeUpdateSpan.parentSpanId, 
+              updatingSpan.spanContext.spanId, 
+              'getSnapshotBeforeUpdate span is not a child of the updating span'
+            );
+            assert.equal(
+              componentDidUpdateSpan.parentSpanId,
+              updatingSpan.spanContext.spanId, 
+              'componentDidUpdate span is not a child of the updating span'
+            );
+
+            assert.strictEqual(exportSpy.args.length, 8, `total number of spans is wrong`);
+          });
+
+          it('spans should have correct name', () => {
+            const forceUpdateSpan: ReadableSpan = exportSpy.args[3][0][0];
+            const renderSpan: ReadableSpan = exportSpy.args[4][0][0];
+            const getSnapshotBeforeUpdateSpan: ReadableSpan = exportSpy.args[5][0][0];
+            const componentDidUpdateSpan: ReadableSpan = exportSpy.args[6][0][0];
+            const updatingSpan: ReadableSpan = exportSpy.args[7][0][0];
+  
+            assert.equal(updatingSpan.name, 'reactLoad: updating', 'updating span has wrong name');
+            assert.equal(forceUpdateSpan.name, 'forceUpdate()', 'forceUpdate span has wrong name');
+            assert.equal(renderSpan.name, 'render', 'render span has wrong name');
+            assert.equal(getSnapshotBeforeUpdateSpan.name, 'getSnapshotBeforeUpdate', 'getSnapshotBeforeUpdate span has wrong name');
+            assert.equal(componentDidUpdateSpan.name, 'componentDidUpdate', 'componentDidUpdate span has wrong name');
+          });
+
+          it('spans should have correct attributes', () => {
+            const spans: [] = exportSpy.args;
+            spans.forEach(element => {
+              const span: ReadableSpan = element[0][0];
+              const attributes = span.attributes;
+              const keys = Object.keys(attributes);
+
+              assert.ok(
+                attributes[keys[0]] !== '',
+                `attributes ${GeneralAttribute.COMPONENT} is not defined for span "${span.name}"`
+              );
+
+              assert.ok(
+                attributes[keys[1]] !== '',
+                `attributes ${AttributeNames.LOCATION_URL} is not defined for span "${span.name}"`
+              );
+
+              assert.ok(
+                attributes[keys[2]] !== '',
+                `attributes ${AttributeNames.REACT_NAME} is not defined for span "${span.name}"`
+              );
+
+              assert.ok(
+                attributes[keys[3]] !== '',
+                `attributes ${AttributeNames.REACT_STATE} is not defined for span "${span.name}"`
+              );
+
+              assert.strictEqual(keys.length, 4, `number of attributes is wrong for span "${span.name}"`);
             });
           });
         });
       });
     });
+
+    describe('when updating AND shouldComponentUpdate returns false', () => {
```

Comment:
minor nit: I see that sometimes you would refer to React methods as `method()`. Maybe consider keeping it consistent here?

Reply:
I was originally doing this to refer to lifecycle methods by name and regular functions (setState, forceUpdate) as a name with brackets to be consistent with this: https://projects.wojtekmaj.pl/react-lifecycle-methods-diagram/
If this doesn't make sense/is confusing I can change it though!